src/frontend/packages/lib_classroom/src/components/DashboardClassroomStudent/index.tsx

```diff
@@ -148,9 +148,23 @@ const DashboardClassroomStudent = ({
   } else {
     // classroom exists but not started
     left = (
-      <DashboardClassroomMessage
-        message={intl.formatMessage(messages.classroomNotStarted)}
-      />
+      <React.Fragment>
+        <Text
+          size="large"
+          weight="bold"
+          color="blue-active"
+          textAlign="center"
+          margin={{ top: 'large' }}
+        >
+          {classroom.title}
+        </Text>
+        <Text color="blue-active" textAlign="center">
+          {classroom.description}
+        </Text>
```

Comment:
This block of code is common between this case and when `starting_at` is set. Maybe you can refactor the code to avoid repetition ?